### DISCUSSION TITLE
Poetry

### DISCUSSION THREAD

**Discussion by pollyannas** [srid/emanote#420] [Q&A] [answered]
Hi, 
I want to post poetry, but the lines are getting merged because of markdown.
What syntax should I use?
Thank you.

**Comment by srid** [comment, marked as answer]
See https://commonmark.org/help/tutorial/03-paragraphs.html

If you append `\` to all but the last line, it should render without putting the lines in the same paragraph. For example,

```markdown
The world?  Moonlit\
Drops shaken\
From the crane's bill.
```

This renders as:

<img width="184" alt="Screenshot 2023-02-07 at 6 48 48 PM" src="https://user-images.githubusercontent.com/3998/217393170-cc7e3039-af41-4658-ae61-7011a25955dc.png">
